src/js/components/Ballot/BallotItem.jsx

```diff
@@ -21,9 +21,9 @@ export default class BallotItem extends Component {
   }
 
   render () {
-    return <div className="ballot-item well well-skinny gutter-top--small">
+    return <div className="ballot-item gutter-top--small">
 
-        <span className="display-name">
+        <span className="display-name h2">
```

Comment:
I was using the Bootstrap class to add some quick styling. Ultimately, I will want to pull heading styles in differently, but it was a quick and easy way to accomplish this styling for the time being.